### PR TITLE
fix #68: Set status bar appearance based on theme

### LINKED ISSUE #68: Fix status bar appearing black in dark theme
While using the dark theme, the phone's status bar appears black, making the information on it illegible. Making it white in dark and amoled (it is completely invisible on amoled mode) themes would fix this issue

Link to image that describes how it currently looks: https://imgur.com/a/byq7gpv

### PR DESCRIPTION
**Pull request by SriMani-7**
## Fix status bar appearing black in dark theme #68

I have implemented a fix that conditionally sets the `WindowInsetsControllerCompat.isAppearanceLightStatusBars` variable based on the app theme.

### CODE REVIEW

**Comment by vishal2376** [comment]
Please Attach Screenshots of both Light and Dark Mode

File: .idea/vcs.xml
Comment (vishal2376) [inline .idea/vcs.xml:1]: Don't include additional files

**Comment by SriMani-7** [comment]
> Please Attach Screenshots of both Light and Dark Mode

Dark theme
Dark theme screenshot

Light theme
Light theme screenshot

Amoled theme
Amoled theme screenshot

**Comment by vishal2376** [comment]
OK, code is working fine.

But 1 line of code doesn't justify 4 commit.
Please do not make extra irrelevant commits

Make single commit of your changes, then i will merge you PR

> If you encounter problem then create a new PR with single commit and don't forget to include issue number and screenshots in description of PR

**Pull request merged**